modules/PersonalBank.py

```diff
@@ -27,35 +29,38 @@ def show_transaction_history(history_of_transactions):
     
         sorted_list = sorted(history_of_transactions, 
                 key = lambda Transaction: Transaction.date)
-        for i in sorted_list:
-            print(i)
+        return sorted_list
```

Comment:
The list should be returned in a separate method called `show_transaction_history()` check again #7